tracing-brave/src/main/java/io/micronaut/tracing/brave/BraveTracerConfiguration.java

```diff
@@ -117,16 +116,6 @@ public void setSampler(@Nullable Sampler sampler) {
         }
     }
 
-    /**
```

Comment:
Removing this should probably be documented in a Breaking Changes section of the docs. I assume we can't just deprecate it because `ErrorParser` is gone.

You can see an example of how I have been doing it here: https://github.com/micronaut-projects/micronaut-serialization/pull/811